### ISSUE TITLE
Getting Fatal Error upon activation

### ISSUE THREAD

**Issue by mintplugins** [RebelCode/edd-bookings#25]
I am trying to help one of your customers for you but I can't install your plugin in my local environment to try and replicate their issue. I have run the composer installation steps but upon activation it is throwing a fatal error:

```
Fatal error: Can't inherit abstract function Aventura\Edd\Bookings\Controller\ControllerInterface::hook() (previously declared abstract in Aventura\Edd\Bookings\Controller\ControllerAbstract) in /Users/philjohnston/Documents/wps/edd2.dev/wp-content/plugins/edd-bookings/includes/Controller/ControllerAbstract.php on line 13
```

**Comment by mintplugins** [comment]
Note to self: Customer ticket to address once plugin is activated without error:
https://secure.helpscout.net/conversation/364849055/59180/

**Comment by mecha** [comment]
@mintplugins What PHP Version are you using?

**Comment by mintplugins** [comment]
It was 5.3

**Comment by mecha** [comment]
Is it more recent than 5.3.9? What you are experiencing is a  PHP bug that was fixed in 5.3.9. If you're interested, here's the [bug report](https://bugs.php.net/bug.php?id=43200).

**Comment by mecha** [comment]
@mintplugins Have you confirmed that the above was the case? If so, let me know and I'll close the issue.
